cpp/docs/DEVELOPER_GUIDE.md

```diff
@@ -1141,3 +1141,22 @@ cuIO is a component of libcudf that provides GPU-accelerated reading and writing
 formats commonly used in data analytics, including CSV, Parquet, ORC, Avro, and JSON_Lines.
 
 // TODO: add more detail and move to a separate file.
+
+# Debugging libcudf
```

Comment:
Question for everyone. 

I would have presumed that this would have gone into the `## Debugging cuDF` section of `CONTRIBUTING.md`. Is that not the correct place as that is more focused on python development?

Reply:
I will move this into `CONTRIBUTING.md`

Reply:
Thanks